src/components/AcceptTermsForm/index.tsx

```diff
@@ -42,7 +42,7 @@ const AcceptTermsForm = () => {
       </p>
 
       <Checkbox
-        checkboxId="acceptTermsCheckbox"
+        checkboxId="input-acceptTerms"
```

Comment:
Just to be consistent that all input field ids start with `input-`. This convention is coming from the Design System.